docs/common-patterns.rst

```diff
@@ -169,7 +166,7 @@ restrictions highly readable.
         }
 
         modifier onlyAfter(uint _time) {
-            if (now < _time) throw;
+            require(now > _time);
```

Comment:
Changing now but wouldn't that signify it allows something to happen also at a given time and not just after a given time.